<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thomas</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyMpdJuke</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/karaoke.png</normaloff>:/res/karaoke.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.565217 rgba(50, 50, 50, 255), stop:1 rgba(100, 100, 100, 255));
font: 14pt &quot;Sans Serif&quot;;
color: rgb(237, 237, 237);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="stateLabel">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Currently playing:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Artist:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curArtist">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>28</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 28pt &quot;Sans Serif&quot;;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curTitle">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>28</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 28pt &quot;Sans Serif&quot;;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="songProgress">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(75, 75, 75);
selection-background-color: rgb(126, 0, 2);

</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="searchBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(75, 75, 75);
font: 20pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>&amp;Search for tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="adminBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(75, 75, 75);</string>
        </property>
        <property name="text">
         <string>&amp;Admin</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QListWidget" name="playlist">
      <property name="styleSheet">
       <string notr="true">selection-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
   </layout>
  </widget>
  <action name="actionUjh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ujh</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>searchBtn</tabstop>
  <tabstop>adminBtn</tabstop>
  <tabstop>playlist</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>adminBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>enterAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>919</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>866</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enterAdmin()</slot>
 </slots>
</ui>
